### DISCUSSION TITLE
[Bug] Network directories are not recognized

### DISCUSSION THREAD

**Discussion by djdookie** [swar/Swar-Chia-Plot-Manager#105] [Q&A]
It seems that network directories are not recognized.

If you put e.g. \\\nas\plots into config as destination_directory, it is not shown in the view.
Do we have to mask such directories somehow?

**Comment by Amneamnius** [comment]
I think this is a known issue. For now you would have to run another script to copy files out from a local drive to your NAS or mask it somehow to appear as a local drive.

**Reply by djdookie** [reply]
Thanks. Yes I just connected my network storage path as drive Z: on all my plotting machines. This works fine.

**Comment by spacewaffles309** [comment]


**Reply by djdookie** [reply]
Yes, a "landing disk" is a good solution to save some time, but you have even more TBW burned of your SSDs. ;)
I have multiple plotting machines, and most of them have no fast drives, just a bunch of older spare drives. In this case, directly copying the files away to the target farmer via 1 GBit Ethernet should be more efficient, as long as not all machines copy via network at the same time.
I guess the start early phase 4 method should have a greater impact in my case.

**Comment by swar** [comment]
What is your config.yaml looking like? Because it does recognize network directories on Windows, but not sure about Linux / Mac. I think it should as well

**Reply by djdookie** [reply]
I am on Windows 10, and in the config
"destination_directory: \\\nas\plots" didn't work while "destination_directory: Z:\" does.
(I connected the Ubuntu Samba share \\\nas\plots to drive Z:)

This tool is really helpful and working great so far!

Is there a good way to use 1 central view and multiple remote managers for multiple plotting machines?
I have seen other plotter tools working with server/client architecture to achieve this.

**Comment by allischalmersman** [comment]
So I have my NAS drive mapped to a drive letter and it's shown in the manager viewer, but number of plots is a "?".  Is this normal?  This occurs on both plotting machines I'm using this tool on.   Great tool BTW!

![image](https://user-images.githubusercontent.com/83618954/117557380-42338400-b040-11eb-823e-1cb5ddc677a4.png)

**Reply by rullywowpcb** [reply]
Having the same issue on a linux system. Network drive is mounted as local directory but I get a question mark as well. Do i just have to wait a while or is this a bug?

Great program, btw...thank you!

![plots_as_questionmark_blur](https://user-images.githubusercontent.com/77423782/117746799-fd902000-b1da-11eb-99af-fd5d674fdcbc.png)

**Comment by tritechnik** [comment]
Same issue here. Mapped to Z:/

**Comment by gsly** [comment]
Same issue on Linux except the drives are mounted locally (i.e. /mnt/enclosureX/driveY/plots) with multiple destinations and all show "?" in plots column.
